lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -201,7 +209,7 @@ describe('manager/npm/post-update/yarn', () => {
       ]);
       expect(fs.readFile).toHaveBeenCalledTimes(expectedFsCalls);
       expect(fs.remove).toHaveBeenCalledTimes(1);
-      expect(res.lockFile).toBe('package-lock-contents');
+      expect(res.lockFile).toBeUndefined();
```

Comment:
Why this field became undefined?

Reply:
![image](https://user-images.githubusercontent.com/8514156/149681287-bf3234ae-f416-47a9-8057-e52e78a79046.png)

When `upgrades` param of function `generateLockFile` contains `isLockFileMaintenance` we should remove lock file. Therefore previous implementation of unit test was not fully correct.

Reply:
I have added `yarn.lock` file content to Fixture of this unit test to be sure it removes